### ISSUE TITLE
Suggestion: Persist closing of Github issue modal throughout session

### ISSUE THREAD

**Issue by geleeroyale** [Giveth/giveth-2#421]
The "Give feedback" notification should replicate the behaviour of the cookies notification and stay closed for the duration of the session.

**Comment by laurenluz** [comment]
@mateodaza just to confirm... you said in the dev call today that the popup, once closed, stays closed for 3 days?

I'm testing now on staging and it didn't even popup at the start of my session. I think it would be good to have it at least popup once at the beginning....

**Comment by mateodaza** [comment]
can you try again logged out or incognito?

**Comment by laurenluz** [comment]
I tested it in incognito and yes, it comes up. but if I close the popup and reload staging in a new tab, it doesn't show up again. I think it would be nice to have it at least once at the start of each session, rather than have it gone for 3 days.

**Comment by WhyldWanderer** [comment]
As a user, if I want to provide feedback but have already closed the pop-up, there is no way to do so.
Can we add a link somewhere to allow for feedback even after the pop up has been closed?

**Comment by geleeroyale** [comment]
I think we have to rework this as well. Once the support widget is ready, I am sure people will prefer it over making Github issues, so *only devs* would use this feature going forward. 

IMO we should go back to the traditional approach that TRACE uses as well (have a GitHub button - ideally - in the header of the DApp)

OR if we want to keep the popup - have the gitcoin button appear once the popup has been closed and disappear in session where the popup is open?

**Comment by WhyldWanderer** [comment]
During the dev call on Wednesday 4/21 we talked about needing a larger conversation around how we will collect user feedback and provide support to users - Added to this epic so that we can discuss it further

**Comment by WhyldWanderer** [comment]
From the community call today, it sounds like we will be discussing this topic further in the governance meeting tomorrow Monday the 26th. 

To decide on a flow for how we will collect and respond to feedback, suggestions, and issues from the user.
